client/post-editor/editor-drawer/index.jsx

```diff
@@ -263,6 +264,14 @@ const EditorDrawer = React.createClass( {
 		);
 	},
 
+	renderCopyPost: function() {
+		if ( 'post' !== this.props.type ) {
```

Comment:
It's limited to post as per trial requirements, but I agree it makes sense for pages too. Just... maybe in another PR!

And yes, after several months of rebasing, it makes more sense to inline it in render, but it didn't back then! :D